### PR TITLE
[feat] Resolve Issue #16 : Wishlist button on cards

### LINKED ISSUE #16: Wishlist button in product cards
It would be great if the product cards shown on Product page has the `Add to Wishlist` btn as well.

Chores:
1. Add Wishlist button to cards

### PR DESCRIPTION
**Pull request by Hamdrive**
Referring to issue #16 , successfully added wishlist button to the product cards in product page of FueledUp™

### CODE REVIEW

File: css/products.css [hunk 1]
```diff
@@ -61,26 +61,51 @@
...
-    footer .grid-cols-2 {
-      grid-template-columns: 0;
-    }
-  
-    .grid-cols-auto {
-      grid-template-columns: repeat(auto-fit, minmax(18rem, 1fr));
-    }
+  .page-content {
+    padding-left: 0;
+    padding-right: 0;
```
Comment (hinanshisuthar) [inline css/products.css]: If you have made variables around padding, prefer to use that.

**Review by hinanshisuthar — COMMENTED** [review]
Liked how you have used variables for giving values to widths, heights, and colors. Shows usability of code.

**Review by shuklahimanshu81 — COMMENTED** [review]
Code looks very clean and also as Hinanshi said good use of variables done. Can be merged.

File: css/products.css (hunk 1 above)
Comment (Hamdrive) [reply css/products.css]: Sure I did create a `size 0` might as well use it atlast!

**Comment by Hamdrive** [comment]
Suggested changes implemented, else no other major issues found.
This fixes issue #16 and is being merged to dev and closed and deleted.

**Pull request merged**